infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/runtime/fsm/ReplicatingState.java

```diff
@@ -108,10 +108,6 @@ public void onEntry(LogReplicationRuntimeState from) {
     public void onExit(LogReplicationRuntimeState to) {
         log.debug("Transition to {} from replicating state.", to.getType());
         switch (to.getType()) {
-            case STOPPED:
```

Comment:
why did we remove this state transition?

Reply:
This isn't doing a state transition...its just checking if the next state is STOPPED, then we were shutting down the sourceManager. This same action is done by the StoppedState.onEntry() as well.
So it was a duplicate action, and I thought its appropriate to remove this one and keep the one in the StopopedState